### DISCUSSION TITLE
Duda con el día de entregas  y Fechas Limites

### DISCUSSION THREAD

**Discussion by D-a-v-i-d-Vargas** [LaunchX-InnovaccionVirtual/CursoIntroPython#221] [Q&A] [answered]
Hola, a todos, tengo dudas y estoy algo confundido por el día que deberiamos de subir nuestras katas, según tengo entendido es el día domingo (13 de frbrero, osea hoy) y el nuevo curso empieza el día lunes 14 de frebrero, pero lo que no me ha quedado muy claro, sí es el ultimo día para poder subir nuestras katas porque como muchos aqui soy nuevo en esto y apenas voy en la kata 6 :( 

P.d. a quíen lea esto gracias y espero que ayude a los demas como a mi me ayudo la sección de dudas para mis katas saludos a todos :)

**Comment by OscarPedroza** [comment]
Hola, las katas se entregan a partir del día de mañana, donde habilitarán el enlace de un formulario para entregar el repositorio con las katas, si no se entrega mañana aun tienes una semana más todavía para poder entregarlo.

En la sesión del viernes se dan detalles del proceso de envio:

https://teams.microsoft.com/l/meetup-join/19%3ameeting_Yjg2N2NlOWYtNTgyNi00Y2JlLWJmOWQtYjAyZDYwYjViODli%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%227b10324a-57d4-43ce-95fe-68f52b498664%22%2c%22IsBroadcastMeeting%22%3atrue%7d

**Comment by JaelDS** [comment, marked as answer]
A partir de mañana comienzan las entregas y hasta el 21 de febrero, me parece, terminan de recibir las katas. Sin embargo mañana comienza el front end a las 8 de la noche. En cuanto mandes tu link, te darán acceso a front end, no importa que entres desfasado. Ve a tu ritmo y tranquilo para que subas los documentos de forma correcta y con ello comiences la siguiente fase.

**Reply by jacobvdz** [reply]
Hola! Y muchas gracias por responder la duda! (Yo también la tenía). Entonces lo de front end cada quien lo comenzará en distintos días dependiendo de cuando terminen la primera fase?

**Reply by ghost** [reply]
Muchas gracias por responder tenía esa misma duda ya que mañana empieza la siguiente misión y a penas le voy agarrando bien la onda a las katas. Éxito, nos veremos pronto en la siguiente misión.

**Reply by jcarlostoso** [reply]
Muchas gracias por la respuesta. andaba con esa duda. Aun no logro terminar la kata 10

**Reply by mikaelzen** [reply]
> Hola! Y muchas gracias por responder la duda! (Yo también la tenía). Entonces lo de front end cada quien lo comenzará en distintos días dependiendo de cuando terminen la primera fase?

Por lo que entendí, el comienzo de Front End es hoy 14 para todos, pero habrá oportunidad de entrar con algunos días de atraso; más lo ideal sería esforzarse para ponerse al corriente cuanto antes.

**Reply by Daniel-Cruz19** [reply]
Que tal compañera, la siguiente clase cuando es?, aun no acabo las katas y se que la primer clase fue hoy, me podrías decir cuando es la siguiente por favor para ponerme al corriente lo mas antes posible, gracias.

**Reply by jfaynel** [reply]
Muchas gracias! quiero ir al dia, pero esto es muy nuevo para mi y batallo un poco, a darle con todo que esta muy interesante y divertido!
